docs/config.md

```diff
@@ -653,14 +654,16 @@ profile_models:
     parameters:
         batch_sizes: 4
 perf_analyzer_flags:
-  input-data: my-input-data-1
   shape:
     - INPUT0:1024,1024
     - INPUT1:1024,1024
 ```
 
 If a model configuration has variable-sized dimensions in the inputs section, 
 then the `shape` option of the `perf_analyzer_flags` option must be specified.
+More information about this can be found in the 
+[Perf Analyzer](https://github.com/triton-inference-server/server/blob/main/docs/perf_analyzer.md#input-data)
+documentation.
```

Comment:
Small nit, but should the word 'documentation' also be included as part of linking text?

Reply:
Fixed!